specs/monitoring_integration.adoc

```diff
@@ -281,23 +281,24 @@ Create a new component called monitoring-integration and implement its functiona
 * Create a new tendrl component called monitoring-integration
 * Create a directory called grafana to place all predefined json files and configuration
   files for grafana.
-* Create a files in grafana directory to configure the grafana environment using
+* Create files in grafana directory to configure the grafana environment using
   predefined jsons.
 * Create a grafana configuration file called grafana.conf.yml
 * create a graphite configuration file called graphite.conf.yml
-* Create a directory called server to create an endpoint for receive grafana
+* Create a directory called server to create an endpoint for receiving grafana
   alert events and convert that alert events into tendrl alert condition and pass the
```

Comment:
Where are we going to store the default values like:
default panel, dashboard, database name, and other details.